qhub/deploy.py

```diff
@@ -459,6 +468,7 @@ def _attempt_tcp_connect(host, port, num_attempts=3, timeout=5):
     }
     ip_or_name = stage_outputs[directory]["load_balancer_address"]["value"]
     host = ip_or_name["hostname"] or ip_or_name["ip"]
+    host = host.strip("\n")
```

Comment:
There appears to be a newline character at the end of `host` for a reason I have yet to figure out.

Reply:
This should be fine. If the newline for some reason doesn't exist in the future, this won't cause any problems.